wagtailio/project_styleguide/templates/patterns/base.html

```diff
@@ -0,0 +1,45 @@
+{% load static wagtailuserbar %}
+
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title>
+            {% block title %}
+                {% if self.seo_title %}{{ self.seo_title }}{% else %}{{ self.title }}{% endif %}
+            {% endblock %}
+            {% block title_suffix %}
+                {% with self.get_site.site_name as site_name %}
+                    {% if site_name %}- {{ site_name }}{% endif %}
+                {% endwith %}
+            {% endblock %}
+        </title>
+        <meta name="description" content="" />
```

Comment:
Question: where will the content for this be pulled from? It looks like the existing site uses: ```{% if page.social_text %}{{ page.social_text }}{% elif page.listing_intro %}{{ page.listing_intro }}{% endif %}```

Reply:
so, in this case, `self` is whatever object (an instance page model in this case) calls it. So `self == page`. Just as an aside, this is set within the page's `get_context()` method, you can see this [here](https://github.com/wagtail/wagtail/blob/dc610366146b2dab04895d7619318d391ba435f1/wagtail/models/__init__.py#L1658), this is called when the page is rendered.

Reply:
Should we add tags within `content=""` then? E.g.

```
<meta name="description" content="{% if page.social_text %}{{ page.social_text }}{% elif page.listing_intro %}{{ page.listing_intro }}{% endif %}" />
```